portfolio/src/main/webapp/contact.html

```diff
@@ -7,7 +7,7 @@
         <script src="script.js"></script>
     </head>
 
-    <body onload="initParam('limit'); initParam('sort'); initParam('searchName'); getComment()">
+    <body onload="initParam('limit'); initParam('sort'); initParam('searchName'); getComments(); isAuth()">
```

Comment:
Since you now have multiple function calls in `onload`, I would replace it with an `init()` that calls those other functions in `script.js`.